corehq/apps/analytics/tasks.py

```diff
@@ -324,21 +325,24 @@ def track_web_user_registration_hubspot(request, web_user, properties):
     )
 
 
-@analytics_task(serializer='pickle', )
-def track_user_sign_in_on_hubspot(webuser, hubspot_cookie, meta, path):
+@analytics_task()
+def track_user_sign_in_on_hubspot(webuser_id, hubspot_cookie, meta):
```

Comment:
Curious how you decided `path` was no longer being used? I still see it used here: https://github.com/dimagi/commcare-hq/blob/7d8e654b47da9a97f10a7bc88bd86f6e3cf9978d/corehq/apps/analytics/signals.py#L74-L75